src/components/molecules/TextView/TextView.tsx

```diff
@@ -62,16 +82,25 @@ const TextView: FC<IProps> = ({ data, segmentText }) => {
 
   return (
     <div className={classes.root}>
-      <Box color={shadowColor} textAlign="center">
+      <Box
+        my={'30px'}
```

Comment:
?

Reply:
you right, that's redundant, i already have margin override in `headerList style class`.